test/e2e/nutanix_client_test.go

```diff
@@ -32,7 +32,7 @@ import (
 	infrav1 "github.com/nutanix-cloud-native/cluster-api-provider-nutanix/api/v1beta1"
 )
 
-var _ = Describe("Nutanix client [PR-Blocking]", func() {
+var _ = Describe("Nutanix client", func() {
```

Comment:
same

Reply:
correct, its not unless you think otherwise